docker/scripts/deprovision.sh

```diff
@@ -263,20 +263,14 @@ if [[ -d /sys/firmware/efi ]]; then
 	done
 fi
 
-# Call firmware script to update components and firmware
-case "$class" in
-baremetal_2a2 | baremetal_2a4 | baremetal_hua)
-	echo "skipping hardware update for oddball aarch64s"
-	;;
-*)
-	set_autofail_stage "running packet-hardware inventory"
-	packet-hardware inventory --verbose --tinkerbell "${tinkerbell}/hardware-components"
-	# Catalog various BIOS feature states (not yet supported on aarch64)
-	if [[ $arch == "x86_64" ]]; then
-		bios_inventory "${id}" "${class}" "${facility}"
-	fi
-	;;
-esac
+# Run packet-hardware inventory to update API components and firmware details
+if ! packet-hardware inventory --verbose --tinkerbell "${tinkerbell}/hardware-components"; then
+	echo "Warning: packet-hardware inventory failed for server ${id} (${class})"
```

Comment:
In general, warnings should go to stderr (but please do stay consistent with what the rest of osie does)